components/layout/SubNav.tsx

```diff
@@ -52,7 +53,9 @@ export const SubNav: React.FC<SubNavProps> = ({ pageType, npmTag }) => {
   return pageType !== PAGE_TYPES.NOT_FOUND ? (
     <MarketingSubNav>
       <SubNavTabs label="Status page nav" items={subNavItems(pageType)} />
+      {/* @ts-ignore-error - TODO */}
```

Comment:
I don't understand why TypeScript was unhappy about these. They're just styled components. 🤷‍♂️ 

I did this to un-block this PR, but it should be addressed for real later.